README.md

```diff
@@ -216,6 +216,30 @@ const User =
     })
 ```
 
+### ID Fields
+
+It is possible to declare a field as an id, wich will take impact on the entity metadata and our glue libs.
```

Comment:
"It is possible to declare a field as an ID. This metadata will be used by glues to enrich the infrastructure interfaces (Database, REST, GraphQL, etc)."